crates/thermocouple-converter/tests/test.rs

```diff
@@ -0,0 +1,41 @@
+use crate::voltage_to_celsius;
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn voltage_to_celsius_test1() {
```

Comment:
Hey, generally tests that are conceptually the same should be under the same test function.